src/dsl/expr.rs

```diff
@@ -165,13 +257,10 @@ impl Expr {
 impl Display for Expr {
     // `f` is a buffer, and this method must write the formatted string into it
     fn fmt(&self, f: &mut Formatter) -> Result {
-        use AggExpr::*;
         use Expr::*;
         match self {
             Alias(expr, name) => write!(f, "{expr} AS {name}"),
-            Agg(agg_expr) => match agg_expr {
-                Sum(expr) => write!(f, "sum({expr})"),
-            },
+            Agg(agg_expr) => f.write_str(&agg_expr.to_string()),
```

Comment:
No need to produce an intermediate String here.
```
Agg(agg_expr) => write!(f, "{agg_expr}"),
```

Reply:
Quick question about this for my own understanding - `write!(f, "{agg_expr}")` is more efficient because it uses `.fmt` instead of `.to_string()` and avoids materializing an intermediate string?